docs/guides/integration-mobile-client-side.md

```diff
@@ -0,0 +1,700 @@
+:::
+ 
+## Pass Generated Token for Bid Stream Use
+
+In your mobile app, if the call to `generateIdentity()` was successful, it returned an identity, which you can then use in the bid stream.
+
+First, the method call to generate the identity:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+UID2Manager.getInstance().generateIdentity()
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+UID2Manager.shared.generateIdentity()
+```
+
+</TabItem>
+</Tabs>
+
+Then, the method call to get the advertising token:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+UID2Manager.getInstance().getAdvertisingToken()
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+UID2Manager.shared.getAdvertisingToken()
+```
+
+</TabItem>
+</Tabs>
+
+If successful, this method call returns a non-null string object such as the following: 
+
+```js
+AgAAAQFt3aNLXKXEyWS8Tpezcymk1Acv3n+ClOHLdAgqR0kt0Y+pQWSOVaW0tsKZI4FOv9K/rZH9+c4lpm2DBpmFJqjdF6FAaAzva5vxDIX/67UOspsYtiwxH73zU7Fj8PhVf1JcpsxUHRHzuk3vHF+ODrM13A8NAVlO1p0Wkb+cccIIhQ==
+```
+
+You can use this identity to pass downstream for sending in the RTB bid stream.
+
+If the `getAdvertisingToken()` method call returns `null`, there was no identity or valid token generated.
+
+Some possible reasons for this, and some things you could do to troubleshoot, are as follows:
+
+- Check to see whether there are any errors from the `generateIdentity()` call.
+- Enable logging to get more information: see [Enable Logging](#enable-logging)
+- It's possible that the identity that was generated has expired. If this happens, you'll need to call the `generateIdentity()` method again: see [Client-Side Integration Example](#client-side-integration-example).
+
+For more information, see [Best Practices on When to Pass DII to the UID2 Mobile SDK](#best-practices-on-when-to-pass-dii-to-the-uid2-mobile-sdk).
+
+## Best Practices on When to Pass DII to the UID2 Mobile SDK
+
+The first time a new user opens the app, no UID2 identity exists. You'll need to call the `generateIdentity()` method, with the DII, to start the token generation:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+UID2Manager.getInstance().generateIdentity()
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+UID2Manager.shared.generateIdentity()
+```
+
+</TabItem>
+</Tabs>
+
+When this method call completes successfully, the advertising token (UID2 token) is available for you to send to the bid stream.
+
+If the UID2 identity stored in local file storage has expired and cannot be refreshed, you must call the `generateIdentity()` method again to generate a new identity and get the resulting UID2 token.
+
+The only exception is if the following Android method/iOS object returns a status of `OPT_OUT`, which means that the DII has been opted out of UID2 and no identity/token should be generated for it:
```

Comment:
_The only exception is if the following Android method/iOS object returns a status of `OPT_OUT`, which means that the DII has been opted out of UID2 and no identity/token should be generated for it, **using the following codes**:_

iOS's is not a method call

Reply:
No fix here per discussion.